### PR TITLE
Added 6 Speakers

### LINKED ISSUE #323: Add speakers Kamesh Sampath, Suraj Narwade, and Aayush Arora
Kamesh Sampath 
His info is on https://eventyay.com/e/b548bcfc/#speakers
This is the photo:
image

Suraj Narwade 
His info is on https://eventyay.com/e/b548bcfc/#speakers
This is the photo:
image

Aayush Arora
His info is on https://eventyay.com/e/b548bcfc/#speakers
The photo's quality is bad.

### PR DESCRIPTION
**Pull request by rishabh19038**
Fixes #323 #324 

Added Speakers: Kamesh Sampath, Suraj Narwade, Aayush Arora, Pronnoy Goswami, Himanshu Singh.
Modified image of Speaker: Harshit Dwivedi in 300x300 dimensions.

Preview: https://adg-vit.github.io/2018.fossasia.org/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3098,6 +3098,119 @@ <h1>Speakers</h1>
...
+                            </div>
+                        </div>
+
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                              <div class="image-holder">
+                                    <img class="background-image" alt="Pronnoy Goswami" src="img/PronnoyGoswami.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="https://www.linkedin.com/in/pronnoygoswami/"><i class="icon fa fa-external-link"></i></a>
```
Comment (simsausaurabh) [inline index.html]: Remove the external link since linkedIn and external are same.

File: index.html [hunk 2]
```diff
@@ -3098,6 +3098,119 @@ <h1>Speakers</h1>
...
+                            </div>
+                        </div>
+
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                              <div class="image-holder">
+                                    <img class="background-image" alt="Himanshu Singh" src="img/HimanshuSingh.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="https://www.linkedin.com/in/himanshusingh2407"><i class="icon fa fa-external-link"></i></a>
```
Comment (simsausaurabh) [inline index.html]: External link of Himanshu Singh is given, but you have added likedIn link of him. Check at eventyay.

File: index.html (hunk 1 above)
Comment (rishabh19038) [reply index.html]: @simsausaurabh Removed.

File: index.html (hunk 2 above)
Comment (rishabh19038) [reply index.html]: @simsausaurabh Himanshu Singh's external link contains LinkedIn profile only at eventyay, so I've removed the External Link Button for him as well. Please review.

**Comment by rishabh19038** [comment]
@simsausaurabh Changes made.

**Comment by hpdang** [comment]
@rishabh19038 could you resolve the conflict plz?

**Comment by rishabh19038** [comment]
@hpdang I've resolved the conflicts. Please merge. Thank You!

**Comment by edenyay** [comment]
Hi @rishabh19038, @simsausaurabh @duttaditya18 @gautamjajoo, can someone add title "Android Dev Lead" for Harshit Dwivedi #324

**Comment by rishabh19038** [comment]
@edenyay Done so!

**Comment by duttaditya18** [comment]
@rishabh19038 Good work!

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Comment by rishabh19038** [comment]
@mariobehling Resolved. Please check. Thanks!

**Pull request closed without merging**